packages/manager/modules/pci/src/projects/project/storages/object-storage/objects/onboarding/translations/Messages_fr_FR.json

```diff
@@ -13,5 +17,9 @@
   "pci_projects_project_storages_objects_onboarding_guides_share-object-temporary-url_title": "Partager un objet avec une adresse temporaire",
   "pci_projects_project_storages_objects_onboarding_guides_share-object-temporary-url_description": "Découvrez comment partager un objet sans fournir vos informations personnelles",
   "pci_projects_project_storages_objects_onboarding_guides_sync-object-containers_title": "Synchroniser des conteneurs d’objets",
-  "pci_projects_project_storages_objects_onboarding_guides_sync-object-containers_description": ""
+  "pci_projects_project_storages_objects_onboarding_guides_sync-object-containers_description": "",
```

Comment:
**nitpick:** Instead of adding empty keys, better to add logic in the controller

Reply:
Okay Anoop will do it.